src/pages/settings/Profile/PersonalDetails/AddressPage.js

```diff
@@ -152,54 +168,53 @@ function AddressPage(props) {
                         maxInputLength={CONST.FORM_CHARACTER_LIMIT}
                     />
                 </View>
-                <View style={styles.mb4}>
-                    <TextInput
-                        inputID="addressLine2"
-                        label={props.translate('common.addressLine', {lineNumber: 2})}
-                        defaultValue={street2 || ''}
-                        maxLength={CONST.FORM_CHARACTER_LIMIT}
-                    />
-                </View>
-                <View style={styles.mb4}>
-                    <TextInput
-                        inputID="city"
-                        label={props.translate('common.city')}
-                        defaultValue={address.city || ''}
-                        maxLength={CONST.FORM_CHARACTER_LIMIT}
+                <View style={styles.formSpaceVertical} />
```

Comment:
This `View` looks hacky. Let's avoid it.

Reply:
This is to fix another reported issue, that the margins weren't correct, and each component was using its own margin, using that `View`, gives me more control.
I tend to avoid margins because it breaks the component isolation, I tend to use "Spacers", I moved into its own function but got feedback that wasn't needed 
https://mxstbr.com/thoughts/margin/
https://javascript.plainenglish.io/stop-using-margin-use-spacer-component-instead-953d9b2dbacc

Reply:
Checking this again. I don't think we use spacers anywhere. It would be better to keep things consist or at least bring this to Slack before moving forward.

cc @mountiny thoughts here?